static/plugins/gtm/index.js

```diff
@@ -0,0 +1,35 @@
+// Inspired by https://github.com/LukasGentele/docusaurus-gtm-plugin (the one we were using)
+//   and https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-tag-manager
+//   But modified to support a custom URL for the tag manager script.
+
+module.exports = function pluginGoogleTagManager(_context, options) {
+  if (process.env.NODE_ENV !== "production") {
+    return null;
+  }
```

Comment:
This will prevent GTM from rendering when running locally. The plugin we're previously using doesn't do this, but it seems like the right thing to do.